impeller/entity/entity_unittests.cc

```diff
@@ -2685,25 +2685,32 @@ TEST_P(EntityTest, AdvancedBlendCoverageHintIsNotResetByEntityPass) {
       << "The ColorBurned texture wasn't allocated (100x100 scales up 2x)";
 }
 
+// glyph_atlas_pipelines_.CreateDefault(
```

Comment:
Nit: Did you mean to leave in this checked in comment?

Reply:
removed